core/src/main/java/hudson/model/Actionable.java

```diff
@@ -134,7 +144,7 @@
      */
     @Nonnull
     public <T extends Action> List<T> getActions(Class<T> type) {
-        List<T> _actions = Util.filter(getActions(), type);
+        List<T> _actions = Util.filter(getPersistedActions(), type);
```

Comment:
I think this should be reverted, so it would still take into account additions from pre-`TransientActionFactory` implementations.

Reply:
right... child may have overrides